src/main/webapp/assets/js/util/brooklyn-utils.js

```diff
@@ -178,7 +178,7 @@ define([
         $.ajax({
             type: "POST",
             dataType: "text",
-            url: "/logout",
```

Comment:
does the endpoint support `POST` ?  in https://github.com/apache/brooklyn-server/pull/41 it says only `GET`